src/test/java/homeworks/POM/Pages/HomePageHmw.java

```diff
@@ -0,0 +1,30 @@
+package homeworks.POM.Pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+
+public class HomePageHmw extends BasePageHmw {
+    By avatarLocator = By.cssSelector("img.avatar");
+    By soundBarPlayLocator =By.cssSelector("[data-testid='sound-bar-play']");
```

Comment:
**I recommend:**  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

```
    private By avatarLocator = By.cssSelector("img.avatar");
    private By soundBarPlayLocator =By.cssSelector("[data-testid='sound-bar-play']");
```